### PR TITLE
Speciality code review

### PR DESCRIPTION
**Pull request by SaraVieira**
## Description - [Trello ticket](https://trello.com/c/tHPVj1HI)

put a short summary of what you did here

## Review template

- [x] checked that this PR resolves the spec provided from trello / this description
- [x] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

File: src/components/Speciality/books.js [hunk 1]
```diff
@@ -44,66 +44,59 @@ const BooksBox = styled.a`
   `}
 `
 
-const BooksSection = ({ speciality }) =>
-  speciality.externalResources.filter(
-    additionalInfo => additionalInfo.type === `Book`
-  ).length ? (
+const BooksSection = ({ books, title }) =>
+  books.length ? (
```
Comment (antoniocapelo) [inline src/components/Speciality/books.js]: probably add a `books && books.length` just to make sure it won't break?

File: src/components/Speciality/books.js [hunk 2]
```diff
@@ -44,66 +44,59 @@ const BooksBox = styled.a`
...
-                </Padding>
-              </BookCol>
-            ))}
+          {books.slice(0, 3).map(externalResource => (
+            <BookCol
+              width={[1, 1, 1, 1 / 2, 1 / 2, 4 / 12]}
+              key={`${externalResource.id}`}
+            >
+              <Padding top={4}>
+                <BooksBox
```
Comment (antoniocapelo) [inline src/components/Speciality/books.js]: can you be a good scout and define 

```
const BooksBox = styled(ExternalAnchor)`

```

?

File: src/components/Speciality/tutorials.js [hunk 3]
```diff
@@ -13,10 +13,8 @@ const TutorialsGrid = styled(Grid)`
   padding-bottom: ${remcalc(18)};
 `
 
-const TutorialsSection = ({ speciality }) =>
-  speciality.externalResources.filter(
-    externalResource => externalResource.type === `Tutorial`
-  ).length > 0 ? (
+const TutorialsSection = ({ tutorials, externalResources }) =>
+  tutorials.length > 0 ? (
```
Comment (antoniocapelo) [inline src/components/Speciality/tutorials.js]: same comment as above

File: src/components/Speciality/tutorials.js [hunk 4]
```diff
@@ -29,26 +27,18 @@ const TutorialsSection = ({ speciality }) =>
...
+                    <ExternalAnchor href={externalResource.link}>
+                      {externalResource.additionalInfo}
+                    </ExternalAnchor>
+                  </CustomisedBulletpoint>
+                ))}
               </ul>
               <Padding top={3}>
-                <StyledLink href={speciality.externalResources[7].link}>
-                  {speciality.externalResources[7].title}
+                <StyledLink href="http://nodetuts.com/">
```
Comment (antoniocapelo) [inline src/components/Speciality/tutorials.js]: same, is this hardcoded?

File: src/utils/generateCaseStudy.js [hunk 5]
```diff
@@ -5,6 +5,7 @@
 */
 
 const generateCaseStudy = caseStudy => {
+  console.log(JSON.stringify(caseStudy))
```
Comment (antoniocapelo) [inline src/utils/generateCaseStudy.js]:
```suggestion
```

File: src/components/Speciality/books.js (hunk 1 above)
Comment (SaraVieira) [reply src/components/Speciality/books.js]: Changed the getExternalType to:
```
  const getExternalType = type =>
    speciality.externalResources.filter(
      additionalInfo => additionalInfo.type === type
    ) || []
```

File: src/components/Speciality/books.js (hunk 2 above)
Comment (SaraVieira) [reply src/components/Speciality/books.js]: Done!

**Comment by SaraVieira** [comment]
Added new issues to card

**Pull request merged**